packages/trader/src/App/__tests__/app.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from '../app';
+import { mockStore } from '@deriv/stores';
+import moment from 'moment';
+import { TStores } from '@deriv/stores/types';
+
+const root_store = {
+    ...mockStore({}),
+    common: {
+        ...mockStore({}).common,
+        server_time: moment(new Date()).utc(),
+    },
+    client: {
+        ...mockStore({}).client,
+        is_landing_company_loaded: true,
+        is_logged_in: false,
+    },
+    modules: {
+        ...mockStore({}).modules,
+        cashier: {
+            general_store: {
+                onMountCommon: jest.fn(),
+                setAccountSwitchListener: jest.fn(),
+            },
+        },
+    },
+};
+
+const mock_ws = {
```

Comment:
let's use camelCase in both files :)
```suggestion
const mockWs = {
```